pkg/services/sqlstore/datasource.go

```diff
@@ -66,11 +66,20 @@ func GetDataSource(query *models.GetDataSourceQuery) error {
 
 func GetDataSources(query *models.GetDataSourcesQuery) error {
```

Comment:
@marefr i figured i could have used this with a tiny modification, let me know what you think